.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+    add a comment in that issue. This will help us to track the need for a feature. If the feature
+    is indeed new, then please open a new issue using the "Feature request"
+    template.
+
+3. Contributing Code
+
+    Please note that NWChemEx is a large project, with many moving pieces. We
+    are also very concerned with performance. This requires planning, and
+    maintaining a unified design. Until you are familiar with the inner workings
+    of NWChemEx and the overall design, we **strongly** recommend chatting with
+    a more established developer **before** spending any extensive time writing
+    code. The best way to do this is by opening a draft pull request (PR) and
+    articulating what you intend to do, and how you intend to do it. While we
+    strive to be an open and welcoming community, the size of the project
+    mandates that we adhere to the unifying design to keep things on track. PRs
+    will need to be aligned with this design before they can be merged.
+
+    That all said, we understand that this may seem daunting for new developers.
+    We also understand it takes time to come to grips with the project. We
+    consider ourselves a welcoming and helpful community and are more than
+    willing to walk you through the process and answer any questions you may
+    have. Just remember we all started somewhere.
+
+    Also if you're just committing a small change, no need to run it by us. Just open the (non-draft) PR with the change!
+
+    For help on creating a pull request see [this](https://nwchemex-project.github.io/.github/resources/github/pull_request.html) tutorial.
+
+## Contributor License Agreement (CLA)
+
+Since the NWChemEx project is developed primarily by the U.S. Department of
+Energy, you will need sign a CLA for legal reasons. When you make a PR, you will
+automatically be prompted to view and sign the CLA (if you have  not already).
+The contents of the CLA are:
+
+```
+By contributing computer software code to the CMakePP project I certify that the
+contribution is being made under the Apache 2.0 license
+(https://www.apache.org/licenses/LICENSE-2.0) and that I have all rights and
+permissions necessary to do so, including authorization from my employer if
+applicable.
+
+NOTICE. This Software was developed under funding from the U.S. Department of
+Energy and the U.S. Government consequently retains certain rights. As such, the
+U.S. Government has been granted for itself and others acting on its behalf a
+paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+reproduce, distribute copies to the public, prepare derivative works, and
+perform publicly and display publicly, and to permit others to do so.
+```
+
+This should be self-explanatory, but if you have questions just ask.
+
+## Code of Conduct
+
+All interactions with the NWChemEx organization are governed by our
+[Code of Conduct](https://github.com/NWChemEx/.github/blob/master/.github/CODE_OF_CONDUCT.md).
```

Comment:
Does it make sense to have this section exposed without the CoC in place?

Reply:
I went ahead and added the CoC to #64 (including un-commenting these lines).